disnake/http.py

```diff
@@ -1174,6 +1175,80 @@ def get_thread_members(self, channel_id: Snowflake) -> Response[List[threads.Thr
         route = Route("GET", "/channels/{channel_id}/thread-members", channel_id=channel_id)
         return self.request(route)
 
+    def create_tag(
+        self,
+        channel_id: Snowflake,
+        name: str,
+        emoji_id: Optional[Snowflake] = None,
+        emoji_name: Optional[str] = None,
+    ) -> Response[channel.ForumChannel]:
+        payload: Dict[str, Any] = {"name": name}
+
+        if emoji_id is not None:
+            payload["emoji_id"] = emoji_id
+
+        if emoji_name is not None:
+            payload["emoji_name"] = emoji_name
+
+        route = Route("POST", "/channels/{channel_id}/tags", channel_id=channel_id)
+        return self.request(route, json=payload)
+
+    def edit_tag(
+        self,
+        channel_id: Snowflake,
+        tag_id: Snowflake,
+        name: str,
+        emoji_id: Snowflake,
+        emoji_name: Optional[str],
+    ) -> Response[channel.ForumChannel]:
+        payload = {
+            "name": name,
+            "emoji_id": emoji_id,
+            "emoji_name": emoji_name,
+        }
+        route = Route(
+            "PUT", "/channels/{channel_id}/tags/{tag_id}", channel_id=channel_id, tag_id=tag_id
+        )
+        return self.request(route, json=payload)
+
+    def delete_tag(self, channel_id: Snowflake, tag_id: Snowflake) -> Response[None]:
+        route = Route(
+            "DELETE", "/channels/{channel_id}/tags/{tag_id}", channel_id=channel_id, tag_id=tag_id
+        )
+        return self.request(route)
+
+    def create_forum_post(
+        self,
+        channel_id: Snowflake,
+        files: Optional[Sequence[File]] = None,
+        reason: Optional[str] = None,
+        **fields: Any,
+    ) -> Response[threads.Thread]:
+        valid_keys = (
+            # Post fields
+            "name",
+            "auto_archive_duration",
+            "rate_limit_per_user",
+            # Message fields
+            "content",
+            "embeds",
+            "embed",
+            "allowed_mentions",
+            "components",
+            "sticker_ids",
+            "attachments",
+            "flags",
+        )
+        payload = {k: v for k, v in fields.items() if k in valid_keys}
+        payload["type"] = 11
+        route = Route("POST", "/channels/{channel_id}/threads", channel_id=channel_id)
+
+        if files is not None:
+            multipart = to_multipart_with_attachments(payload, files)
+            return self.request(route, form=multipart, files=files, reason=reason)
```

Comment:
Please include a param 'has_message' to True, per https://canary.discord.com/channels/927603005627437097/927603006579568723/962075735835672686